src/app/models/propTypes/index.js

```diff
@@ -41,27 +41,25 @@ export const imagePropTypes = {
 };
 
 export const videoPropTypes = {
-  model: shape({
-    blocks: arrayOf(
-      shape({
-        locator: string,
-        blocks: arrayOf(
-          // raw video
-          shape({
-            model: shape({
-              isLive: bool,
-              duration: string,
-              locator: string.isRequired,
-            }),
+  blocks: arrayOf(
+    shape({
+      locator: string,
+      blocks: arrayOf(
+        // raw video
+        shape({
+          model: shape({
+            isLive: bool,
```

Comment:
Shouldn't these all be required for the BBC player to work properly?

Reply:
Possibly, but that is outside the remit of this PR really. Intend to open more issues to review props of Image and Video, and move them to the new format.